docs/app/components/modal/modal-demo.component.html

```diff
@@ -0,0 +1,252 @@
+<ng-template #content let-close="close" let-dismiss="dismiss" let-data="data">
+    <hc-modal>
+        <hc-modal-header [dismiss]="dismiss"> Modal Title </hc-modal-header>
+        <hc-modal-body> Data: {{data}}</hc-modal-body>
+        <hc-modal-footer>
+            <button hc-button color="tertiary" (click)="dismiss()"> Cancel </button>
+            <button hc-button color="primary" (click)="close()"> OK </button>
+        </hc-modal-footer>
+    </hc-modal>
+</ng-template>
+
+<div class="demo-content">
+    <h1>Modal</h1>
+    <h6>Last updated {{lastModified | date:'longDate'}}</h6>
+
+    <hc-tab-set direction="horizontal">
+        <hc-tab tabTitle="Overview">
+
+            <hc-tile>
+                <h5>About</h5>
+                <p>A modal dialog is a window that forces the user to interact with it before they can go back to using the
+                    parent application.</p>
+            </hc-tile>
+
+            <hc-tile>
+                <h5>Example</h5>
+                <button hc-button color="primary" (click)="open()"> Open Modal from Component </button>
+                <button hc-button color="secondary" (click)="openTemplate(content)"> Open Modal from Template </button>
```

Comment:
Can you add `class="demo-button"` to both of these buttons?  That just makes sure that if the buttons end up getting stacked on top of each other on a smaller screen, there is some vertical space between them.

Reply:
Done